dspace-api/src/main/java/org/dspace/authenticate/LDAPAuthentication.java

```diff
@@ -503,6 +503,7 @@ protected String getDNOfUser(String adminUser, String adminPassword, Context con
                     } else {
                         searchName = ldap_provider_url + ldap_search_context;
                     }
+                    @SuppressWarnings("BanJNDI")
```

Comment:
These are suppressed because the sysadmin. has control of the directory and can ensure safety.